fhir/parser_test.go

```diff
@@ -16,7 +16,7 @@ func Test_ParseConformanceStatement(t *testing.T) {
 	if err != nil {
 		t.Errorf("Error in sending mock request in test %s", err.Error())
 	}
-	var capabilityStatement = ParseCapabilityStatement(&resp)
+	var capabilityStatement, _ = ParseCapabilityStatement(&resp)
```

Comment:
why is the error being ignored here?

Reply:
I didn't bother checking for error since this is a very basic test and the asserted FHIR version likely wouldn't be correct in the event of a parsing error. -- Will add a check/failed test condition for `err`